addons/point_of_sale/static/src/app/screens/payment_screen/payment_status/payment_status.xml

```diff
@@ -2,42 +2,32 @@
 <templates id="template" xml:space="preserve">
 
     <t t-name="point_of_sale.PaymentScreenStatus">
-        <section class="paymentlines-container bg-view p-3 mb-1 border-bottom">
-            <div t-if="props.order.get_paymentlines().length === 0" class="paymentlines-empty">
-                <div class="total text-center py-2 text-success">
-                    <t t-esc="totalDueText" />
-                </div>
-                <div class="message text-center">
-            Please select a payment method.
-                </div>
+        <t t-if="props.order.get_paymentlines().length == 0">
+            <div class="message text-center py-4 fs-4">
+        Please select a payment method 
             </div>
-
-            <t t-else="">
+        </t>
+        <t t-else="">
+            <section class="paymentlines-container p-3 border-bottom">
                 <div class="payment-status-container d-flex flex-column-reverse flex-lg-row justify-content-between fs-2">
-                    <div>
-                        <div class="payment-status-remaining">
+                    <t t-if="this.props.order.get_due() >= 0 and props.order.get_change() === 0">
```

Comment:
you might want to use `gte` here.
[reference](https://github.com/odoo/owl/blob/master/doc/reference/templates.md#:~:text=it%20can%20use%20a%20few%20special%20operators%20to%20avoid%20using%20symbols%20such%20as%20%3C%2C%20%3E%2C%20%26%20or%20%7C.%20This%20is%20useful%20to%20make%20sure%20that%20we%20still%20write%20valid%20XML.)